pkg/streamingpromql/engine_test.go

```diff
@@ -2981,8 +2977,6 @@ func TestCompareVariousMixedMetricsVectorSelectors(t *testing.T) {
 		for _, function := range []string{"rate", "increase", "changes", "resets", "deriv", "irate", "idelta", "delta", "deriv", "stddev_over_time", "stdvar_over_time"} {
 			expressions = append(expressions, fmt.Sprintf(`%s(series{label=~"(%s)"}[45s])`, function, labelRegex))
 			expressions = append(expressions, fmt.Sprintf(`%s(series{label=~"(%s)"}[1m])`, function, labelRegex))
-			expressions = append(expressions, fmt.Sprintf(`sum(%s(series{label=~"(%s)"}[2m15s]))`, function, labelRegex))
```

Comment:
Rationale: we already extensively test `sum` in `TestCompareVariousMixedMetricsAggregations`.

Reply:
I would like to keep these. They test reusing pools that have been used by the range-vector function.

Reply:
Do we need both `sum` test cases?

Don't forget we also have `TestConcurrentQueries` that should cover pooled slices being reused as well.

Reply:
one is probably sufficient. I'd suggest keeping this one (2m15s).

Reply:
Done, I've restored that in c4b2c57.